app/models/venue.rb

```diff
@@ -1,2 +1,9 @@
 class Venue < ActiveRecord::Base
+  def display_t_accessible
+    if t_accessible
```

Comment:
Also, convention suggests that methods which return either a true or False (like 't_accessible') end in a `?`. (This totally works, though - and I love the pattern of extracting into a smaller method like this!)